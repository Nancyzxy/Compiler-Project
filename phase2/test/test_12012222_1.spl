struct Car{
    int cost;
    float velocity;
    int aa;
    int aaa;
    float aaaa;
};
struct Car{
    int vmefifty;
};

int main(){
    struct Car cars[14];
    struct Car carb;
    int a = 1;
    int flag = 0;
    cars[1].cost.cost = 15;
    cars[2].cost = 425.6;
    cars[2].velocity = 5.14;  
    if(carb == cars[1].cost){
        
    }
    return 0;
}