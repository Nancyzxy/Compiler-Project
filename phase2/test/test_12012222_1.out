Error type 15 at Line 8: redefine struct: Car
Error type 13 at Line 17: accessing with non-struct variable
Error type 5 at Line 18: unmatching type on both sides of assignment
Error type 7 at Line 20: binary operation on non-number variables
